### ISSUE TITLE
Config for a gun's Ammo Capacity and Bullet Item

### ISSUE THREAD

**Issue by TheRebelT** [cybercat-mods/HWG#91]
A config for each gun to alter it's ammo capacity before needing to reload. Possibly a config to change the bullet item that the gun consumes in order to reload (Testing of this would need to be done before it can be decided whether or not it'll be supported)

**Comment by TheRebelT** [comment]
There's a crash:
I'm not sure which mod is causing the crashing but I'm using the lastest HWG 1.4.17, SmartBrain 1.12, azurelib 1.0.18 pre 2. And fabric api 0.87.0

Crash Report: https://pastebin.com/KhpTjWCb

Sorry if this is in the wrong place, wasn't sure If I should make a new issue or use this one. Since this hasn't been closed yet, I presumed it was still being worked on
